### PR TITLE
Find The Number

### LINKED ISSUE #4956: [New game]: Find The Number
### 🎮 Game Request

Game logic and basic description

- HTML, CSS, and Javascript are used in the creation of this project.
- The player will have 10 attempts to try their luck and find out the right number 
- If they are able to find it they win .

### Point down the features

Game points

- Random number generation between 1 and 100.
- User has 10 attempts to guess the number.
- Displays previous guesses and remaining attempts.
- Still working on the game will soon finish up .
- On guessing the right Number the User will win the Game.

- also working on its UI.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by AshishPandey04**
## PR Description 📜

I have added a new game Find The Number. I have also improve user experience by Allowing user to restart the game . Have written a Standard javascript code which is error free and gives user nice experince.
Fixes #4956 

<hr>
 
## Mark the task you have completed ✅

✅I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
 ✅ I have performed a self-review of my own code or work.
✅ I have commented my code, particularly in hard-to-understand areas.
✅ | My changes generates no new warnings.
 ✅I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
✅I have added screenshot for website preview in assets/images 
✅ I have added entries for my game in GameZone's README.md
✅ I have added README.md in my Game folder 
✅ I have added working video of the game in README.md (optional)
✅I have specified the respective issue number for which I have requested the new game.
✅ I have added my Game to gamesData.json file and it is loading properly on main website as intended.

<hr>

## Add your screenshots(Optional) 📸
Find_the_Number

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @AshishPandey04 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by AshishPandey04** [comment]
@kunjgit  @ishita-43 pls review the PR and let me know if any change is required

File: README.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ This repository also provides one such platforms where contributers come over an
 
 | [Master Typing](https://github.com/kunjgit/GameZone/tree/main/Games/Master_Typing)                               | [Treasure Hunt](https://github.com/Antiquely3059/GameZone/tree/main/Games/Treasure%20Hunt)                               | [Virtual Pet](https://github.com/Antiquely3059/GameZone/tree/main/Games/Virtual_Pet)                    | [MazeRunner](https://github.com/kunjgit/GameZone/tree/main/Games/MazeRunner)                                 | [Ping_Pong_Singleplayer](https://github.com/kunjgit/GameZone/tree/main/Games/Ping_Pong_Singleplayer)               | [Madlibs](https://github.com/AaryanManghnani/GameZone/tree/main/Games/Madlibs)                                                                                                            |
 | [Bulls_And_Cows_New](https://github.com/kunjgit/GameZone/tree/main/Games/Bulls_And_Cows_New)
+|[Find The Number](https://github.com/AshishPandey04/GameZone/tree/newBranch/Games/Find_the_Number)
```
Comment (ishita-43) [inline README.md:113]: Change this repository name to the kunjgit as other specified to make it workable on the site

**Review by ishita-43 — COMMENTED** [review]
Please make the following change

File: README.md (hunk 1 above)
Comment (AshishPandey04) [reply README.md]: done

**Review by ishita-43 — APPROVED** [review]
Pr approved

**Comment by AshishPandey04** [comment]
@kunjgit  Sir pls merge the PR

**Comment by kunjgit** [comment]
Thank you  @AshishPandey04 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**